package.json

```diff
@@ -134,7 +134,9 @@
       "@typescript-eslint/no-explicit-any": "off",
       "@typescript-eslint/camelcase": "off",
       "@typescript-eslint/interface-name-prefix": "off",
-      "@typescript-eslint/no-unused-vars": "warn",
+      "@typescript-eslint/no-unused-vars": [
+        "warn", {"args": "all", "argsIgnorePattern": "^_"}
+      ],
```

Comment:
I was seeing `Warning: eslint: '_indent' is defined but never used.` for things like `streamsPrologue(_indent: string)`.

Reply:
yeah we need to tweak eslint settings

Reply:
We needed this in APIX too. Thanks for fixing.